### ISSUE TITLE
Question about ProtoMAML

### ISSUE THREAD

**Issue by xugy16** [Nithin-Holla/MetaWSD#3]
Really appreciate for the shared code and it is really helpful.

I have a question about  SeqMetaModel's Protomaml Setting.

1. in this setting, init_weights and init_bias are initilized by ProtoVec. 
2. ProtoVec is calculated by self.learner(for example, bert)
3. then self.learner is the computing graph.

proto_grads = torch.autograd.grad(loss, [p for p in self.learner.parameters() if p.requires_grad])
meta_grads = [mg + pg for (mg, pg) in zip(meta_grads, proto_grads)]

The meta_grads should not be added pg, because pg already in the graph by ProtoVec.

Really appreciate for your reply.
